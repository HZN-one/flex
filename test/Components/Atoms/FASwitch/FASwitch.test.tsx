/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";

import { FASwitch } from "@Atoms";

import { render } from "@testing-library/react";

describe("FASwitch", () => {
  it("should match snapshot", () => {
    const { container } = render(<FASwitch testID="switch-example" />);
    expect(container).toMatchSnapshot();
  });
});
