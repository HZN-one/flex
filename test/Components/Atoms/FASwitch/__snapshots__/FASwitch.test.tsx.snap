// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FASwitch should match snapshot 1`] = `
<div>
  <span
    class="MuiSwitch-root MuiSwitch-sizeMedium css-julti5-MuiSwitch-root"
  >
    <span
      class="MuiSwitch-switchBase MuiSwitch-colorPrimary MuiButtonBase-root MuiSwitch-switchBase MuiSwitch-colorPrimary PrivateSwitchBase-root css-5ryogn-MuiButtonBase-root-MuiSwitch-switchBase"
      data-testid="switch-example"
    >
      <input
        class="MuiSwitch-input PrivateSwitchBase-input css-1m9pwf3"
        type="checkbox"
      />
      <span
        class="MuiSwitch-thumb css-jsexje-MuiSwitch-thumb"
      />
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </span>
    <span
      class="MuiSwitch-track css-1yjjitx-MuiSwitch-track"
    />
  </span>
</div>
`;
